### ISSUE TITLE
Request: Add `wp post duplicate`

### ISSUE THREAD

**Issue by jgraup** [wp-cli/entity-command#141]
It would be great to include a command that is almost identical to [`wp post create`](https://developer.wordpress.org/cli/commands/post/create/) except all the default values are pulled from an existing post (including meta data) and any additional commands will override those defaults.

```
wp post duplicate 123 --post_title="A Different Title"
```

**Comment by gitlost** [comment]
Sounds good to me @jgraup !

**Comment by schlessera** [comment]
If the command is almost identical, it might make more sense to implement this as an extension of `wp post create` instead.

Maybe something like this:
```
wp post create --from-post=123
```

**Comment by sagarnasit** [comment]
I created a new command that takes id of existing post and generates a new post with metadata. Also added other options to override defaults. I don't know what kind of testing I should do and how. can anybody help?

eg. wp post **duplicate** 21

**Comment by schlessera** [comment]
@sagarnasit I'd prefer to have this functionality be an extension of the existing `wp post create` command, as the functionality is almost identical.

**Comment by aaronjorbin** [comment]
I think this can be closed due to #154

**Comment by schlessera** [comment]
Fixed via #154
